admin_guide/scheduler.adoc

```diff
@@ -61,7 +61,7 @@ These predicates can be configured by the user to tweak their functioning.  They
 ----
 **_LabelsPresence_** checks whether a particular node has a certain label defined or not, regardless of its value.
 ----
-{"name" : "ZoneRequired", "argument" : {"serviceAffinity" : {"labels" : ["zone"]}}}
+{"name" : "ZoneRequired", "argument" : {"labels" : ["retiring"], "presence" : true}}}
```

Comment:
Yes, there is an extra `}` in there after the change.

However, to make the example more meaningful, I would suggest that you change the presence from "true" to "false". With presence set to "true", it currently reads as: "Consider ONLY those nodes that have the `retiring` label present." Instead, we want the opposite and hence the "false" value is required.